index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link href="styles.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="hdiv">
+        <header class ="head">Uthman</header>
+        <i class="fa-solid fa-bars menu-bar"></i> 
+    </div> 
+    <section class="section1">
```

Comment:
- [ Required ] Please consider laying out the elements in this section using `flexbox` please note that this is a project requirement. You can do this using the `display: flex`  and `flex-direction: column` CSS rules.

![Screenshot from 2022-12-03 03-13-03](https://user-images.githubusercontent.com/16392046/205411831-5199a635-8cbd-4aff-afc2-2199fae49cab.png)